### PR TITLE
Clean cache after upgrades

### PR DESCRIPTION
**Pull request by keharper**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, instructions for cleaning cache will be provided wherever the reader is instructed to upgrade versions or modules.

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

File: guides/v2.3/get-started/create-integration.md [hunk 1]
```diff
@@ -277,6 +277,10 @@ In Production mode, you may receive a message to 'Please rerun Magento compile c
 
     <code>bin/magento setup:di:compile</code>
 
+3. Run the following command to clean the configuration cache.
+
+    <code>bin/magento cache:clean config</code>
```
Comment (lorikrell) [inline guides/v2.3/get-started/create-integration.md]: Same comment from before on ticks vs `<code>`?

File: guides/v2.3/comp-mgr/cli/cli-upgrade.md [hunk 2]
```diff
@@ -167,7 +167,13 @@ Updating the metadata in `composer.json` file is entirely superficial, not funct
 composer update
 ```
 
-### Clear caches and generated content
+### Clean the Magento configuration cache
+
```
Comment (lorikrell) [inline guides/v2.3/comp-mgr/cli/cli-upgrade.md]: Would it be worth adding a tidbit of info here regarding clearing the cache after upgrades?

File: guides/v2.1/frontend-dev-guide/themes/admin_theme_apply.md [hunk 3]
```diff
@@ -54,16 +54,16 @@ In `<your_module_dir>/etc/adminhtml/di.xml` add the following (replace the place
...
     <!-- Admin theme. End -->
 </config>
 ```
 
 ## Update components to actually apply the Admin theme
 
-For your changes to take effect, you need to update Magento components. For this, 
-run the `bin/magento setup:upgrade` command in your command line. If prompted, also run `bin/magento setup:di:compile`.
+For your changes to take effect, you need to update Magento components. For this,
+run the `bin/magento setup:upgrade` command in your command line. If prompted, also run `bin/magento setup:di:compile`. Then also run `bin/magento cache:clean config` to clear the configuration cache.
```
[the remark this answers is not in the document]
Comment (keharper) [reply guides/v2.1/frontend-dev-guide/themes/admin_theme_apply.md]: My goal is to add the information in whatever format makes sense given the material around it. If the source material were a procedure, I would have added a step. This is not a primary source of information about setup:upgrade, so I won't try to improve the source.

File: guides/v2.1/install-gde/install/cli/dev_update-magento.md [hunk 4]
```diff
@@ -36,4 +36,8 @@ To update the Magento software if you're a contributing developer:
 
 5.	Update the Magento database:
 
-		php <your Magento install dir>/bin/magento setup:upgrade
+		<your Magento install dir>/bin/magento setup:upgrade
```
[the remark this answers is not in the document]
Comment (keharper) [reply guides/v2.1/install-gde/install/cli/dev_update-magento.md]: This topic relies on indentation for code samples.

File: guides/v2.3/comp-mgr/cli/cli-upgrade.md (hunk 2 above)
Comment (keharper) [reply guides/v2.3/comp-mgr/cli/cli-upgrade.md]: Done. I think that was a copy and paste error.

File: guides/v2.1/get-started/create-integration.md [hunk 5]
```diff
@@ -246,6 +246,10 @@ In Production mode, you may receive a message to 'Please rerun Magento compile c
 
     <code>bin/magento setup:di:compile</code>
 
+3. Run the following command to clean the configuration cache.
+
+   <code>bin/magento cache:clean config</code>
```
[the remark this answers is not in the document]
Comment (keharper) [reply guides/v2.1/get-started/create-integration.md]: Matching what I'm given

File: guides/v2.1/frontend-dev-guide/themes/admin_theme_apply.md [hunk 6]
```diff
@@ -54,16 +54,16 @@ In `<your_module_dir>/etc/adminhtml/di.xml` add the following (replace the place
...
                  <item name="adminhtml" xsi:type="string">%Your_vendor_dir%/%your_theme_code%</item> <!-- Example: "Magento/backend" -->
              </argument>
-         </arguments> 
+         </arguments>
     </type>
     <!-- Admin theme. End -->
 </config>
 ```
 
 ## Update components to actually apply the Admin theme
```
[the remark this answers is not in the document]
Comment (keharper) [reply guides/v2.1/frontend-dev-guide/themes/admin_theme_apply.md]: Since I'm dealing with mass changes, editing the doc is beyond the scope of this PR.

**Pull request merged**